security/security-design/fn-datasafe-dbaudit-to-oci-logging/README.md

```diff
@@ -0,0 +1,146 @@
+# Data Safe Audit DB to OCI Logging

```

Comment:
Add a section that lists the environments this has been tested on, e.g., not on Windows.

Reply:
Added section test env.